### PR TITLE
Bugfix: Fix up a previous attempt to quote correctly

### PR DESCRIPTION
**Pull request by tamiko**
In reference to #3742

### CODE REVIEW

**Comment by tamiko** [comment]
OK. Let's wait for the travis instance.

**Comment by kronbichler** [comment]
This does indeed fix the MPI tests and the tester is happy. I'm merging.

File: cmake/scripts/run_test.sh [hunk 1]
```diff
@@ -27,10 +27,18 @@ set -u
...
-NUMDIFF_EXECUTABLE="$4"
-DIFF_EXECUTABLE="$5"
-COMPARISON_FILE="$6"
+NUMDIFF_EXECUTABLE="$3"
+DIFF_EXECUTABLE="$4"
+COMPARISON_FILE="$5"
+shift 5
+
+save () {
+  for i do printf %s\\n "$i" | sed "s/'/'\\\\''/g;1s/^/'/;\$s/\$/' \\\\/" ;
```
Comment (tjhei) [inline cmake/scripts/run_test.sh]: a comment what this magic is would have been nice.

Comment (drwells) [inline cmake/scripts/run_test.sh]: that is a pretty frightening sed command.

**Pull request merged**